lib/validic/response.rb

```diff
@@ -11,5 +13,23 @@ def initialize(summary, response)
         @attributes = response
       end
     end
+
+    def next
+      extend Validic::REST::Utils
```

Comment:
spoke with @jaysonlarin and tried to debug myself. we need to break the circular reference of `Response -> REST::Utils -> Response` in order for this to work correctly